### PR TITLE
#162949043 Pagination for articles

### PR DESCRIPTION
**Pull request by Paulvitalis200**
#### What does this PR do?

Enables the articles page to have pagination.

#### Description of Task to be completed?
- Create a method for getting active page
- Use Semantic React pagination to do pagination

#### How should this be manually tested?
1. Navigate to a folder where you would prefer to test the repository.

2. Create a directory
 ```$```  ```mkdir myproject```

3. Navigate to the directory "myproject"
 ```$```   ```cd myproject```
4. Clone the author's haven repository
  ```$```   ```git clone https://github.com/andela/ah-robotics-frontend.git```
5. Change the directory into the "ah-robotics-fronted"
  ```$```   ```cd ah-robotics-frontend```

6.  Checkout to the branch ft-pagination-162949043
 ```$```   ```git checkout ft-pagination-162949043```

7. Install dependancies from the package.json
 ```$```   ```npm install``` to install all the modules listed in the `package.json` file.

8. Start the application in development mode
  ```$```   ```npm start``` to run the app in the development mode 

The application opens http://localhost:3000 in the browser.
Change the URL on the browser to;
 ```
https://ah-robotics-frontend-sta-pr-18.herokuapp.com
```

9. Scroll down to the bottom of the page to access the pagination links. Click on a specific page to change the page being viewed.

###  Running Tests
```
npm test
```

#### What are the relevant pivotal tracker stories?
[#162949043](https://www.pivotaltracker.com/n/projects/2235171/stories/162949043)

### Screenshot
Image showing the pagination
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,14 +18,16 @@ class App extends Component {
...
           <NavigationComponent />
           <Switch>
-            <Route path="/" component={HomeView} exact />
             <Route path="/login" component={LoginView} exact />
             <Route path="/register" component={RegisterView} />
             <Route path="/verify/:token" component={VerifyRegistrationView} />
             <Route path="/resetpassword" component={ResetPasswordView} />
             <Route path="/reset_password/:token" component={NewPasswordView} />
+            <Route path="/" component={ArticlesView} exact />
+            <Route path="/login" component={LoginView} exact />
```
Comment (sharkdevs) [inline src/App.js]: This Login route is a duplicate of the one above.

**Review by sharkdevs — COMMENTED** [review]
The tests are passing fine and I was able to seamlessly test the pagination. However, consider the change below. Regards.

File: src/App.js (hunk 1 above)
Comment (Paulvitalis200) [reply src/App.js]: Thanks for the observation. I have fixed this

**Review by Teatoller — APPROVED** [review]
@Paulstar200 ,
I have followed the instructions you have enumerated with ease and managed to experience the feature you put up working brilliantly.
The tests are also passing.

Regards
Steven Ennis

**Review by johnwayodi — COMMENTED** [review]
nice work @Paulstar200  It is nice seeing that the articles are now paginated.

**Review by waracci — COMMENTED** [review]
I have been able to click through the pages and the links work as expected.

**Review by Collins33 — APPROVED** [review]

**Pull request merged**